Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:erbium
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY . /usr/src/app
+
+EXPOSE 8080
+
+CMD ["yarn", "start"]
```

Comment:
@jackvanb So this is just using Node to serve the files and respond to API requests. I believe the best practice is to use nginx to server static files and to be used as a proxy for security and performance. Does this help? https://medium.com/swlh/dockerizing-a-react-application-with-docker-and-nginx-19e88ef8e99a

Then should look into using PM2 so that makes sure the node processes are running and will automatically restart them if they die: https://pm2.io/docs/runtime/integration/docker/

Reply:
Will look into it.